src/monodroid/jni/monodroid-glue.cc

```diff
@@ -10,7 +10,10 @@
 #include <errno.h>
 #include <limits.h>
 
+#if defined (APPLE_OS_X)
```

Comment:
What about Linux? :)

Reply:
The only reason it's needed *at all* is for `MonodroidRuntime::get_my_location()`, which on macOS uses `dladdr()`.  Linux doesn't use `dladdr()`, and thus it isn't needed.